generators/client/files-react.js

```diff
@@ -66,7 +66,6 @@ const files = {
         'config/constants.ts',
         'config/dayjs.ts',
         'config/axios-interceptor.ts',
-        { file: 'config/devtools.tsx', method: 'processJsx' },
```

Comment:
Cleanup this file at https://github.com/jhipster/generator-jhipster/blob/834581e11706bc20456e6e43aee837b69c5f35c0/generators/cleanup.js#L307

Reply:
good catch